blueprints/datamodel.tex

```diff
@@ -355,10 +465,33 @@ \subsubsection{Status}
 	Servers.
 \end{itemize}
 
+\paragraph{Tenant}
+
+\paragraph{Vanity Hostnames}
+
+\subsubsection{DNS-Routed Properties}
+
+\paragraph{Bypass TTL}
+
+\paragraph{DNS TTL}
+An unsigned integer that defines the Time-To-Live (TTL) of DNS responses from the Traffic Router for this Delivery Service's routing, in seconds.
+
+\paragraph{Max Records}
+
+\subsubsection{HTTP-Routed Properties}
+
+\paragraph{Additional Response Headers}
+
+\paragraph{Consistent Hashing Regular Expression}
+
+\paragraph{Logged Request Headers}
+
+\paragraph{Significant Query Parameters}
```

Comment:
See above comment about promoting the cachekey concept.

Reply:
This doesn't have anything to do with caching, it's the "Consistent Hashing Query Parameters". Maybe the property needs a different name, but the only other one I could come up with was "Query Parameters Significant for Routing" which is a pretty long name.

Reply:
Or think about it as overlapping subsets of cachekey config that spans both ATS and TR.  You'd never configure TR to do one thing while ATS to do another (assuming you weren't off in the weeds of ATS power).

Reply:
Yeah, one would hope those line up.